client_push_pre_go17_test.go

```diff
@@ -0,0 +1,46 @@
+// +build !go1.7
+
+package apns2_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
+)
+
+func TestClient_PushWithCtx_WithTimeout(t *testing.T) {
```

Comment:
Any chance we could change this to TestClientPushWithCtxWithTimeout just for consistency